src/index.html

```diff
@@ -7,6 +7,287 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="Moyo Logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+            Apple
+            </a>
```

Comment:
```suggestion
            <a
              href="#apple"
              class="nav__link is-active"
            >
              Apple
            </a>
```
indentation. check all similar cases